README.md

```diff
@@ -22,7 +24,16 @@ console.log(results);
 > Hint: Assuming there is a document with a name of "Bob", why does `results` not contain an author model on the second line?
 
 ```js
-// Your answer...
+// You would want to have the author model set outside the method and then set the above find method as a function and pass in bob as a parameter:
+
+var AuthorModel = Schema.AuthorModel
+
+function findByName(bob) {
+  AuthorModel.findOne({name: "Bob"})
+  console.log(results)
+}
+
+
 ```
```

Comment:
The issue is that `.findOne` is asynchronous, so you need to make sure that you call `console.log(results)` in a promise or callback to ensure it happens after an Author has been found and returned from the database.